### PR TITLE
Ubuntu 20.04 install article

### LINKED ISSUE #17638: The docs need to be made for 20.04
On this page https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1910
It's only mention the 19.10, There has to be a page for 20.04.

If you replace the 19.10 with 20.04 the [path](https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb) will still not work.

[Edit by gewarren to add doc details.]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b5cbe58d-ad05-60fd-47d7-0d9ccfeecde4
* Version Independent ID: c9773e9b-dbc1-cb7b-b303-287cf8bd354c
* Content: [Install .NET Core on Ubuntu 19.10 package manager - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1910)
* Content Source: [docs/core/install/linux-package-manager-ubuntu-1910.md](https://github.com/dotnet/docs/blob/master/docs/core/install/linux-package-manager-ubuntu-1910.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### LINKED ISSUE #18094: wget syntax is wrong
The troubleshooting section shows the command `wget O- https://packages.microsoft.com/keys/microsoft.asc`. That syntax is incorrectly (specifically the `O-` switch), I believe it should be `wget -O - https://packages.microsoft.com/keys/microsoft.asc`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b5cbe58d-ad05-60fd-47d7-0d9ccfeecde4
* Version Independent ID: c9773e9b-dbc1-cb7b-b303-287cf8bd354c
* Content: [Install .NET Core on Ubuntu 19.10 package manager - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1910#feedback)
* Content Source: [docs/core/install/linux-package-manager-ubuntu-1910.md](https://github.com/dotnet/docs/blob/master/docs/core/install/linux-package-manager-ubuntu-1910.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Once the .NET Core Ubuntu 20.04 support is released, this PR should be accepted.

- Added article for 20.04
- Added entry to switcher
- Updated TOC

Fixes #17638
Fixes #18094

Depends on #18165

### CODE REVIEW

**Comment by Thraka** [comment]
@dagood OK redo and on hold until I'm given word.

**Comment by dagood** [comment]
@rbhanda @leecow can you please share how to best track when the release with Ubuntu 20.04 support is going out, to coordinate with this PR?

/cc @dleeapho @NikolaMilosavljevic

File: docs/core/install/linux-package-manager-ubuntu-1910.md
Comment (dagood) [inline docs/core/install/linux-package-manager-ubuntu-1910.md:98]: I'd expect this to conflict with https://github.com/dotnet/docs/pull/18141, which fixes it with slightly different (?) syntax. This PR is more consistent with the other pages, though.

**Review by dagood — APPROVED** [review]

**Comment by Thraka** [comment]
@rbhanda @leecow Is .NET Core all ready for 20.04? Can I merge this? (once i figure out why the build keeps failing...)

**Comment by rbhanda** [comment]
.NET Core has been updated and tested for 20.04. We can publish the document now

**Pull request merged**
